packages/jupyter-ai/ui-tests/README.md

```diff
@@ -12,35 +12,38 @@ in [jupyter_server_test_config.py](./jupyter_server_test_config.py).
 
 The default configuration will produce video for failing tests and an HTML report.
 
-## Run the tests
+> There is a new experimental UI mode that you may fall in love with; see [that video](https://www.youtube.com/watch?v=jF0yA-JLQW0).
 
 > All commands are assumed to be executed from the root directory
 
+## Run the tests
+
 To run the tests, you need to:
 
 1. Compile the extension:
 
 ```sh
-jlpm install
-jlpm build:prod
+./scripts/install.sh
```

Comment:
Doesn't this run the `jlpm build` as well?

Reply:
`./scripts/install.sh` does not exactly run the `jlpm build` but it is not necessary. Removed `jlpm build`